docs/reference/supported-environments.md

```diff
@@ -47,9 +47,9 @@ We recommend running Camunda 8 Self-Managed in a Kubernetes environment. We prov
 
 ### Deployment options
 
-With the right configuration, Camunda 8 Self-Managed can be deployed on any [Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/#benefits) distribution (cloud or on-premise). However, we officially test and support a specific list of platforms.
+Camunda 8 self-managed supports the following [CNCF Certified Kubernetes disctributions](https://www.cncf.io/training/certification/software-conformance/#benefits). Some distributions include custom patches or upstream source code changes. Camunda will not provide support for issues arising from these changes.
```

Comment:
I feel this paragraph is a bit out of place: it talks about Kubernetes, but then below we again talk about Kubernetes (vanilla/cloud/OS) and additional deployment methods (Docker, manual), confusing the user potentially.

I suggest have something like:

```
### Deployment options

#### Kubernetes / OpenShift

Camunda 8 self-managed supports ...

- [Stock Kubernetes](/self-managed/setup/install.md)
- [Cloud service providers](/self-managed/setup/install.md) [recommended]
  ...

#### Docker 

...

#### Manual (JAR)

...

```

wdyt? that way all k8s stuff is grouped and is less confusing. also bonus points for making the high-level installation method headers (can link to these)

Reply:
There is a typo above: "disctributions"